test/metabase/query_processor/middleware/results_metadata_test.clj

```diff
@@ -103,7 +106,8 @@
                          :native   {:query "SELECT ID, NAME, PRICE, CATEGORY_ID, LATITUDE, LONGITUDE FROM VENUES"}})
       (get-in [:data :results_metadata])
       (update :checksum class)
-      round-all-decimals'))
+      round-all-decimals'
+      (->> (tu/round-fingerprint-cols [:columns]))))
```

Comment:
Is this needed? Won't `round-all-decimals` convert anything it finds? Also if this does stay, I'd suggest a refactor as `->` and a nested `->>` can be confusing.

Reply:
round-all-decimals rounds to 2, round-fingerprint-cols has some special logic to round :q1 and :q3 more aggressively as they are order-dependent and not all our engines return data in the same order.

Reply:
FWIW I think it's easier/clearer to use `as->` instead of nesting threading macros